pkg/api/admin/openshiftcluster.go

```diff
@@ -48,7 +49,7 @@ type OpenShiftClusterProperties struct {
 	IngressProfiles                 []IngressProfile        `json:"ingressProfiles,omitempty"`
 	Install                         *Install                `json:"install,omitempty"`
 	StorageSuffix                   string                  `json:"storageSuffix,omitempty"`
-	RegistryProfiles                []RegistryProfile       `json:"registryProfiles,omitempty"`
+	RegistryProfiles                []RegistryProfile       `json:"registryProfiles,omitempty" mutable:"true"` // Not actually mutable! We discard it in ToInternal, this just means that any changes do not cause static validation to fail.
```

Comment:
I don't understand this change (even with the comment). Are you saying that there was a bug in the existing code? If so - could you please submit a separate PR fixing only this issue? Side effects of this change are not clear to me from the `+1,576 −172` diff.